### DISCUSSION TITLE
Comment arrêter le téléchargment automatique des vidéo?

### DISCUSSION THREAD

**Discussion by kamess85** [aclap-dev/video-downloadhelper#385] [Help and Q&A]
### Your question

Il y a des vidéos que ne souhaitent pas télécharger. Car lorsque je regarde une vidéo sur youtube ou une plateforme l'application télécharge automatiquement la video.

### Addon & CoApp version and details

Il y a des vidéos que ne souhaitent pas télécharger. Car lorsque je regarde une vidéo sur youtube ou une plateforme l'application télécharge automatiquement la video.

**Comment by jcfrog** [comment]
bonjour, des videos se téléchargent sans aucune intervention de votre part?
